docs/source/cli/model-training-inference/distributed/sagemaker.rst

```diff
@@ -370,187 +369,52 @@ Using ``Processor`` arguments the above example would become:
-```````````````````````````````````````````````
-First, use the following command to generate a Compose YAML file for the Link Prediction training on OGB-MAG graph.
-
-.. code-block:: bash
-
-    python3 generate_sagemaker_docker_compose.py \
-            --aws-access-key <<AWS_ACCESS_KEY>> \
-            --aws-secret-access-key <AWS_SECRET_ACCESS_KEY> \
-            --aws-session-token <AWS_SESSION_TOKEN> \
-            --num-instances 3 \
-            --image <GRAPHSTORM_DOCKER_IMAGE> \
-            --graph-data-s3 s3://<PATH_TO_DATA>/ogbn_mag_lp_3p \
-            --yaml-s3 s3://<PATH_TO_TRAINING_CONFIG>/map_lp.yaml \
-            --model-artifact-s3 s3://<PATH_TO_SAVE_TRAINED_MODEL> \
-            --graph-name ogbn-mag \
-            --task-type link_prediction \
-            --num-layers 1 \
-            --fanout 10 \
-            --hidden-size 128 \
-            --backend gloo \
-            --batch-size 128
-
-The above command will create a Docker Compose file named ``docker-compose-<task-type>-<num-instances>-train.yaml``, which we can then use to launch the job.
-
-As our Docker Compose will use a Docker network, named ``gsf-network``, for inter-container communications, users need to run the following command to create the network before luanch Docker Compose.
-
-.. code-block:: bash
-
-    docker network create "gsf-network"
+.. code:: bash
 
-Then, use the following command to run the Link Prediction training on OGB-MAG graph.
+    pip install 'sagemaker[local]' --upgrade
 
-.. code-block:: bash
+second, ensure your local mode configuration includes a high shared memory size for the
+launched local containers:
 
-    docker compose -f docker-compose-link_prediction-3-train.yaml up
+.. code:: yaml
 
-Running the above command will launch 3 instances of the image, configured with the command and env vars that emulate a SageMaker execution environment and run the ``sagemaker_train.py`` script.
+    local:
+        local_code: true # Using everything locally
+        region_name: "us-west-2" # Name of the region
+        container_config:
+            shm_size: "32G" # Set this according to your available memory
 
-.. Note:: The containers actually interact with S3, so the provided AWS assess key, security access key, and session token should be valid for access S3 bucket.
+This is necessary as GraphStorm uses shared memory for in-memory graph storage.
 
-Run GraphStorm on Docker Compose for Inference
-```````````````````````````````````````````````
-The ``generate_sagemaker_docker_compose.py`` can build Compose file for the inference task with the same arguments as for training, and in addition, but add a new argument, ``--inference``. The below command create the Compose file for the Link Prediction inference on OGB-MAG graph.
-
-.. code-block:: bash
+Set the environment variable ``USE_SHORT_LIVED_CREDENTIALS=1`` if running on EC2 and
+you would like to use the session credentials instead of EC2 Metadata Service credentials:
```

Comment:
Do users still need to activate their aws credentials? If so, we need to point it out here.

Reply:
Add a doc link to `session credentials` in case users do not know what it is.